x-pack/plugins/lens/public/shared_components/visual_options_popover/missing_values_option.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonGroup, EuiFormRow, EuiIconTip, EuiSuperSelect, EuiText } from '@elastic/eui';
+import {
+  FittingFunction,
+  fittingFunctionDefinitions,
+} from '../../xy_visualization/fitting_functions';
+import { ValueLabelConfig } from '../..';
+
+export interface MissingValuesOptionProps {
+  valueLabels?: ValueLabelConfig;
+  fittingFunction?: FittingFunction;
+  onValueLabelChange: (newMode: ValueLabelConfig) => void;
+  onFittingFnChange: (newMode: FittingFunction) => void;
+  isValueLabelsEnabled?: boolean;
+  isFittingEnabled?: boolean;
+}
+
+const valueLabelsOptions: Array<{
+  id: string;
+  value: 'hide' | 'inside' | 'outside';
+  label: string;
+  'data-test-subj': string;
+}> = [
+  {
+    id: `value_labels_hide`,
+    value: 'hide',
+    label: i18n.translate('xpack.lens.xyChart.valueLabelsVisibility.auto', {
+      defaultMessage: 'Hide',
+    }),
+    'data-test-subj': 'lnsXY_valueLabels_hide',
+  },
+  {
+    id: `value_labels_inside`,
+    value: 'inside',
+    label: i18n.translate('xpack.lens.xyChart.valueLabelsVisibility.inside', {
+      defaultMessage: 'Show',
+    }),
+    'data-test-subj': 'lnsXY_valueLabels_inside',
+  },
+];
+
+export const MissingValuesOptions: React.FC<MissingValuesOptionProps> = ({
+  onValueLabelChange,
+  onFittingFnChange,
+  valueLabels,
+  fittingFunction,
+  isValueLabelsEnabled = true,
+  isFittingEnabled = true,
+}) => {
+  const valueLabelsVisibilityMode = valueLabels || 'hide';
+
+  return (
+    <>
+      {isValueLabelsEnabled ? (
```

Comment:
nit: You can use `isValueLabelsEnabled  &&` syntax here to get rid of the `: null`